src/common/options.cc

```diff
@@ -5309,8 +5309,20 @@ std::vector<Option> get_mds_options() {
     .set_description(""),
 
     Option("mds_cache_size", Option::TYPE_INT, Option::LEVEL_ADVANCED)
-    .set_default(100000)
-    .set_description(""),
+    .set_default(0)
+    .set_description("maximum number of inodes in MDS cache (<=0 is unlimited)"),
+
+    Option("mds_cache_memory_limit", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
IMO this should be LEVEL_BASIC (and have a long_description) as it controls something outside-world-facing that many (most?) users will want to adjust.

Reply:
Thanks for the suggestion! I've updated the description.